test/python/http_recorder.py

```diff
@@ -0,0 +1,285 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Utilities (based on VCRpy) to record remote requets and allow testing offline/cached."""
+
+import json
+from vcr.persisters.filesystem import FilesystemPersister
+from vcr import VCR
+
+
+class IdRemoverPersister(FilesystemPersister):
+    """
+    IdRemoverPersister is a VCR persister. This is, it implements a way to save and load cassettes.
+    This persister in particular inherits load_cassette from FilesystemPersister (basically, it
+    loads a standard cassette in the standard way from the FS). On the saving side, it removes some
+    fields in the JSON content of the responses for dummy values.
+    """
+
+    @staticmethod
+    def get_responses_with(string2find, cassette_dict):
```

Comment:
Non-blocking but these auxiliary static methods could be auxiliary module-level private functions.

Reply:
Nit: `string2find` -> `string_to_find`.

Reply:
8d67900

Reply:
about the static method. What module-level private is better?